content/posts/2025-02-26-whats_new_1_8.md

```diff
@@ -0,0 +1,295 @@
+
+With Packet Translation, a few xlate-related columns are added.  The first row with the service shows the translated namespace and pod for the source and destination (Figure 2).
+
+![Flows table with Packet Translation]({page.image('whats-new-1-8/flows-after_xlate.png')})
+Figure 2: Flows table with Packet Translation
+
+To enable this feature in FlowCollector, enter `oc edit flowcollector` and configure the following in the **ebpf** section:
+
+```
+spec:
+  agent:
+    ebpf:
+      sampling: 1  # recommended so all flows are observed
+      features:
+      - PacketTranslation
+```
+
+<p style="text-align: center">Listing 1: Enable Packet Translation feature in FlowCollector</p>
+
+To create a basic Nginx web server, enter the following commands on the command line.  I created two of them, one with the namespace "chiefs" and another with namespace "eagles".
+
+```
+oc new-project chiefs
+oc adm policy add-scc-to-group anyuid system:authenticated
+oc create deployment nginx --image=nginx
+oc expose deployment/nginx --port=80 --target-port=80
+oc expose svc/nginx
+```
+
+<p style="text-align: center">Listing 2: Create an Nginx web server</p>
+
+
+### eBPF Resource Reduction
+
+In an ideal world, observability would take zero resources, but since that's not realistic, the goal is to use the least amount of resources as possible, particularly in CPU and memory usage.
+
+The great news is that 1.8 made some significant resource savings in the eBPF area.  Testing showed that in a 25-node and 250-node cluster scenarios, the savings range from 40% to 57% compared to 1.7.  Before you get too excited, this is eBPF only so when you look at the overall Network Observability, the savings are closer to 11% to 25%, which is still impressive.  Obviously, your savings will vary depending on your environment.
+
+There were two major changes made to the eBPF Agent, which is the component that collects and aggregates flows in Network Observability.  The way hash maps were used and the algorithm to handle concurrency were changed to split this up into two hash maps, one for collecting and processing data and another for enriching the data with Kubernetes information.  Ultimately, this approach uses less memory and CPU cycles than having a hash map per CPU.  Further savings in CPU and memory usage were achieved also due to the hash map change, by doing more de-duplication in reducing the hash map key, based on 12 tuples to essentially 5.  De-duplication is the process of removing copies of the same packet that the eBPF Agent sees on different interfaces as it traverses through the network.
+
+## Developer and Technology Preview Features
+
+The rest of the features are non-GA, meaning they should not be used in a production environment.  Network Events is a Technology Preview feature and the rest are Developer Preview features.
+
+1. Network Events
+2. eBPF Flow Filtering Enhancements
+3. UDN Observability
+4. ePBF Manager Support
+
+
+### Network Events
```

Comment:
While "Network Events" is the correct term, I would also mention network policies directly in the title, because I think that's what speaks really to the users. Maybe like: "Network Policies monitoring with Network Events" ?

Reply:
Named it "Network Events for Monitoring Network Policies" since I want it to begin with "Network Events".